iftk/helpers/eleven_stream.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import AsyncIterator
+from typing import Optional
+from elevenlabs.client import ElevenLabs
+
+
+async def ElevenStream(
+    sentences: AsyncIterator,
+    eleven_client: ElevenLabs,
+    messages: Optional[dict] = None,
+    voice: Optional[str] = "Jessica",
+) -> AsyncIterator:
```

Comment:
Can we type hint the response with whatever type EleventClient returns?
```suggestion
) -> AsyncIterator[???]:
```